### PR TITLE
Added sample models

### PR DESCRIPTION
**Pull request by shehadak**
This PR adds a sample of neural network models including DenseNet, EfficientNet, GoogLeNet, Inception v3, ResNet, and Vision Transformer using the Brain-Score 2.0 format. This PR aims to get users up to speed for testing their benchmarks on a series of models and provides example models for 

The full list of models is being added in #498.

### CODE REVIEW

File: brainscore_vision/models/alexnet/model.py [hunk 1]
```diff
@@ -1,28 +1,41 @@
...
+    url = {http://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf}
+}"""
 
 LAYERS = ['features.2', 'features.5', 'features.7', 'features.9', 'features.12',
           'classifier.2', 'classifier.5']
 
 
 def get_model():
-    model = torchvision.models.alexnet(pretrained=True)
+    device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
```
Comment (mschrimpf) [inline brainscore_vision/models/alexnet/model.py:26]: I think this should rather be in the wrapper/activations helper (which has to also move stimuli to gpu)

File: brainscore_vision/models/alexnet/test.py [hunk 2]
```diff
@@ -2,12 +2,14 @@
 from pytest import approx
 
 from brainscore_vision import score
+from brainscore_vision.utils import seed_everything
+seed_everything(42)
```
Comment (mschrimpf) [inline brainscore_vision/models/alexnet/test.py:6]: move inside methods where needed. `pytest` does not guarantee sequential test execution for instance.

But I also don't see what exactly we have to seed here. Everything should be deterministic:
* the model is uniquely specified
* the benchmarks have no randomness to them

So why do we need to seed?

File: brainscore_vision/models/alexnet/test.py [hunk 3]
```diff
@@ -2,12 +2,14 @@
...
 
 
 @pytest.mark.private_access
```
Comment (mschrimpf) [inline brainscore_vision/models/alexnet/test.py:9]:
```suggestion
```

File: brainscore_vision/models/alexnet/test.py [hunk 4]
```diff
@@ -2,12 +2,14 @@
...
 @pytest.mark.memory_intense
 @pytest.mark.parametrize("model_identifier, benchmark_identifier, expected_score", [
-    ("alexnet", "MajajHong2015.IT-pls", approx(0.508, abs=0.0005)),
+    ("alexnet", "MajajHong2015public.IT-pls", approx(0.5048, abs=0.0005)),
```
Comment (mschrimpf) [inline brainscore_vision/models/alexnet/test.py:12]:
```suggestion
    pytest.param("alexnet", "MajajHong2015.IT-pls", approx(0.508, abs=0.0005), marks=[pytest.mark.private_access]),
    pytest.param("alexnet", "MajajHong2015public.IT-pls", approx(0.5048, abs=0.0005)),
```

Comment (mschrimpf) [reply brainscore_vision/models/alexnet/test.py:12]: let's not throw away tests

File: brainscore_vision/models/crossvit_pretrained/model.py [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+ssl._create_default_https_context = ssl._create_unverified_context
+
+BIBTEX = ""
+LAYERS = [
+    "blocks.1.blocks.1.0.norm1",
+    "blocks.1.blocks.1.4.norm2",
+    "blocks.1.blocks.1.0.mlp.act",
+    "blocks.2.revert_projs.1.2",
+]
+# Description of Layers:
```
Comment (mschrimpf) [inline brainscore_vision/models/crossvit_pretrained/model.py:23]: where do these come from?

File: brainscore_vision/models/crossvit_pretrained/model.py [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+def load_preprocess_custom_model(image_filepaths, image_size, **kwargs):
+    images = load_images(image_filepaths)
+    images = preprocess_images(images, image_size=image_size, **kwargs)
+    return images
+
+
+def get_model():
+    # Generate Model
+    device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
```
Comment (mschrimpf) [inline brainscore_vision/models/crossvit_pretrained/model.py:40]: as for alexnet, move to wrapper

File: brainscore_vision/models/crossvit_pretrained/model.py [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+    # Load Model and create necessary methods
+    # init the model and the preprocessing:
+    preprocessing = functools.partial(
+        load_preprocess_custom_model, image_size=224)
+    # get an activations model from the Pytorch Wrapper
+    activations_model = PytorchWrapper(
+        identifier="cv_18_dagger_408_pretrained", model=model, preprocessing=preprocessing
+    )
+
+    wrapper = activations_model
```
Comment (mschrimpf) [inline brainscore_vision/models/crossvit_pretrained/model.py:54]:
```suggestion
    wrapper = PytorchWrapper(
        identifier="cv_18_dagger_408_pretrained", model=model, preprocessing=preprocessing
    )
```

File: brainscore_vision/models/crossvit_pretrained/test.py [hunk 8]
```diff
@@ -0,0 +1,18 @@
+import pytest
+from pytest import approx
+
+from brainscore_vision import score
+from brainscore_vision.utils import seed_everything
+seed_everything(42)
+
+
+@pytest.mark.private_access
```
Comment (mschrimpf) [inline brainscore_vision/models/crossvit_pretrained/test.py:9]:
```suggestion
```

File: brainscore_vision/models/crossvit_pretrained/test.py [hunk 9]
```diff
@@ -0,0 +1,18 @@
+import pytest
+from pytest import approx
+
+from brainscore_vision import score
+from brainscore_vision.utils import seed_everything
+seed_everything(42)
```
Comment (mschrimpf) [inline brainscore_vision/models/crossvit_pretrained/test.py:6]: as above, move into respective functions

File: brainscore_vision/models/densenet/model.py [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+    assert net in net_constructors, f"Could not find DenseNet network: {net}"
+    model = net_constructors[net](pretrained=True)
+    return [layer for layer, _ in model.named_modules()
+            if layer in include_layer_names]
+
+
+def get_model(net):
+    assert net in net_constructors, f"Could not find DenseNet network: {net}"
+
+    device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
```
Comment (mschrimpf) [inline brainscore_vision/models/densenet/model.py:43]: move to wrapper

File: brainscore_vision/models/densenet/test.py [hunk 11]
```diff
@@ -0,0 +1,31 @@
+import logging
+import sys
+
+import pytest
+from pytest import approx
+
+from brainscore_vision import score
+from brainscore_vision.utils import seed_everything
+
+seed_everything(42)
```
Comment (mschrimpf) [inline brainscore_vision/models/densenet/test.py:10]: move into function

File: brainscore_vision/models/densenet/test.py [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+from pytest import approx
+
+from brainscore_vision import score
+from brainscore_vision.utils import seed_everything
+
+seed_everything(42)
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+
+
+@pytest.mark.private_access
```
Comment (mschrimpf) [inline brainscore_vision/models/densenet/test.py:14]:
```suggestion

```

File: brainscore_vision/models/crossvit_pretrained/test.py [hunk 13]
```diff
@@ -0,0 +1,18 @@
...
+
+@pytest.mark.private_access
+@pytest.mark.memory_intense
+@pytest.mark.parametrize("model_identifier, benchmark_identifier, expected_score", [
+    ("cv_18_dagger_408_pretrained",
+     "MajajHong2015public.IT-pls", approx(0.5362, abs=0.0005)),
+])
+def test_score(model_identifier, benchmark_identifier, expected_score):
+    actual_score = score(model_identifier=model_identifier, benchmark_identifier=benchmark_identifier,
+                         conda_active=True)
```
Comment (mschrimpf) [inline brainscore_vision/models/crossvit_pretrained/test.py:17]: I can see having to set this flag every time being confusing for users. @kvfairchild can we simplify this?

File: brainscore_vision/models/efficientnet/model.py [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+    assert net in net_constructors, f"Could not find EfficientNet network: {net}"
+    model = net_constructors[net](pretrained=True)
+    return [layer for layer, _ in model.named_modules()
+            if layer in include_layer_names]
+
+
+def get_model(net):
+    assert net in net_constructors, f"Could not find EfficientNet network: {net}"
+
+    device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
```
Comment (mschrimpf) [inline brainscore_vision/models/efficientnet/model.py:45]: move into wrapper

File: brainscore_vision/models/efficientnet/test.py [hunk 15]
```diff
@@ -0,0 +1,35 @@
...
+import sys
+
+import pytest
+from pytest import approx
+
+from brainscore_vision import score
+from brainscore_vision.utils import seed_everything
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+seed_everything(42)
```
Comment (mschrimpf) [inline brainscore_vision/models/efficientnet/test.py:11]: move into function

File: brainscore_vision/models/efficientnet/test.py [hunk 16]
```diff
@@ -0,0 +1,35 @@
...
+
+
+@pytest.mark.private_access
```
Comment (mschrimpf) [inline brainscore_vision/models/efficientnet/test.py:14]:
```suggestion
```

File: brainscore_vision/models/googlenet/model.py [hunk 17]
```diff
@@ -0,0 +1,43 @@
...
+# picked all high-level blocks
+LAYERS = [
+    "conv1", "maxpool1", "conv2", "conv3", "maxpool2", "inception3a", "inception3b", "maxpool3",
+    "inception4a", "inception4b", "inception4c", "inception4d", "inception4e", "maxpool4", "inception5a",
+    "inception5b", "avgpool", "fc",
+]
+
+
+def get_model():
+    device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
```
Comment (mschrimpf) [inline brainscore_vision/models/googlenet/model.py:26]: move into wrapper

File: brainscore_vision/models/googlenet/test.py [hunk 18]
```diff
@@ -0,0 +1,28 @@
...
+import sys
+
+import pytest
+from pytest import approx
+
+from brainscore_vision import score
+from brainscore_vision.utils import seed_everything
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+seed_everything(42)
```
Comment (mschrimpf) [inline brainscore_vision/models/googlenet/test.py:11]: move into function/delete

File: brainscore_vision/models/inception_v3/model.py [hunk 19]
```diff
@@ -0,0 +1,48 @@
...
+    "Mixed_6e",
+    "Mixed_7a",
+    "Mixed_7b",
+    "Mixed_7c",
+    "avgpool",
+]
+
+
+def get_model():
+    device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
```
Comment (mschrimpf) [inline brainscore_vision/models/inception_v3/model.py:40]: move into wrapper

File: brainscore_vision/models/inception_v3/test.py [hunk 20]
```diff
@@ -0,0 +1,28 @@
...
+import sys
+
+import pytest
+from pytest import approx
+
+from brainscore_vision import score
+from brainscore_vision.utils import seed_everything
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+seed_everything(42)
```
Comment (mschrimpf) [inline brainscore_vision/models/inception_v3/test.py:11]: move into function

File: brainscore_vision/models/inception_v3/test.py [hunk 21]
```diff
@@ -0,0 +1,28 @@
...
+
+
+@pytest.mark.private_access
```
Comment (mschrimpf) [inline brainscore_vision/models/inception_v3/test.py:14]:
```suggestion
```

File: brainscore_vision/models/pixels/model.py [hunk 22]
```diff
@@ -20,7 +21,8 @@ def identifier(self):
     def identifier(self, value):
         self._extractor.identifier = value
 
-    def __call__(self, *args, **kwargs):  # cannot assign __call__ as attribute due to Python convention
+    # cannot assign __call__ as attribute due to Python convention
```
Comment (mschrimpf) [inline brainscore_vision/models/pixels/model.py:24]:
```suggestion
    # Note that we cannot assign __call__ as attribute due to Python convention, 
    # so we cannot set `self.__call__ = self._extractor` in the `__init__` and 
    # instead have to explicitly implement the function here.
```

File: brainscore_vision/models/pixels/test.py [hunk 23]
```diff
@@ -2,12 +2,14 @@
 from pytest import approx
 
 from brainscore_vision import score
+from brainscore_vision.utils import seed_everything
+seed_everything(42)
 
 
 @pytest.mark.private_access
```
Comment (mschrimpf) [inline brainscore_vision/models/pixels/test.py:9]:
```suggestion
```

File: brainscore_vision/models/pixels/test.py [hunk 24]
```diff
@@ -2,12 +2,14 @@
...
 @pytest.mark.memory_intense
 @pytest.mark.parametrize("model_identifier, benchmark_identifier, expected_score", [
-    ("pixels", "MajajHong2015.IT-pls", approx(0.0153, abs=0.0005)),
+    ("pixels", "MajajHong2015public.IT-pls", approx(0.0764, abs=0.0005)),
```
Comment (mschrimpf) [inline brainscore_vision/models/pixels/test.py:12]:
```suggestion
    pytest.param("pixels", "MajajHong2015.IT-pls", approx(0.0153, abs=0.0005), marks=[pytest.mark.private_access]),
    pytest.param("pixels", "MajajHong2015public.IT-pls", approx(0.0764, abs=0.0005)),
```

File: brainscore_vision/models/pixels/requirements.txt [hunk 25]
```diff
@@ -0,0 +1,2 @@
+torch
+torchvision
```
Comment (mschrimpf) [inline brainscore_vision/models/pixels/requirements.txt:2]: I don't think we need those here

File: brainscore_vision/models/resnet/model.py [hunk 26]
```diff
@@ -0,0 +1,67 @@
...
+                       for block, block_units in enumerate(units) for unit in range(block_units)]
+        return layers
+    model = net_constructors[net](pretrained=True)
+    return [layer for layer, _ in model.named_modules()][1:]
+
+
+def get_model(net):
+    assert net in net_constructors, f"Could not find ResNet network: {net}"
+
+    device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
```
Comment (mschrimpf) [inline brainscore_vision/models/resnet/model.py:50]: move to wrapper

File: brainscore_vision/models/resnet/test.py [hunk 27]
```diff
@@ -0,0 +1,20 @@
+import pytest
+from pytest import approx
+
+from brainscore_vision import score
+from brainscore_vision.utils import seed_everything
+seed_everything(42)
+
+
+@pytest.mark.private_access
```
Comment (mschrimpf) [inline brainscore_vision/models/resnet/test.py:9]:
```suggestion
```

File: brainscore_vision/models/squeezenet/model.py [hunk 28]
```diff
@@ -0,0 +1,49 @@
...
+LAYERS = [
+    'classifier',
+    *{f'features.{i}' for i in range(13)}
+]
+
+
+def get_model(net):
+    assert net in net_constructors, f"Could not find SqueezeNet network: {net}"
+
+    device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
```
Comment (mschrimpf) [inline brainscore_vision/models/squeezenet/model.py:32]: move to wrapper

File: brainscore_vision/models/squeezenet/test.py [hunk 29]
```diff
@@ -0,0 +1,29 @@
+import logging
+import sys
+
+import pytest
+from pytest import approx
+
+from brainscore_vision import score
+from brainscore_vision.utils import seed_everything
+
+seed_everything(42)
```
Comment (mschrimpf) [inline brainscore_vision/models/squeezenet/test.py:10]: move into function / delete

File: brainscore_vision/models/resnet/test.py [hunk 30]
```diff
@@ -0,0 +1,20 @@
+import pytest
+from pytest import approx
+
+from brainscore_vision import score
+from brainscore_vision.utils import seed_everything
+seed_everything(42)
```
Comment (mschrimpf) [inline brainscore_vision/models/resnet/test.py:6]: move into function/delete

File: brainscore_vision/models/squeezenet/test.py [hunk 31]
```diff
@@ -0,0 +1,29 @@
...
+from pytest import approx
+
+from brainscore_vision import score
+from brainscore_vision.utils import seed_everything
+
+seed_everything(42)
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+
+
+@pytest.mark.private_access
```
Comment (mschrimpf) [inline brainscore_vision/models/squeezenet/test.py:14]:
```suggestion
```

File: brainscore_vision/models/vision_transformer/model.py [hunk 32]
```diff
@@ -0,0 +1,59 @@
...
+
+    # Reverse layer order to avoid channel_x affecting transformer layers
+    return [layer for layer, _ in model.named_modules()
+            if layer in include_layer_names][::-1]
+
+
+def get_model(net):
+    assert net in net_constructors, f"Could not find ViT network: {net}"
+
+    device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
```
Comment (mschrimpf) [inline brainscore_vision/models/vision_transformer/model.py:42]: move into wrapper

File: brainscore_vision/models/vision_transformer/test.py [hunk 33]
```diff
@@ -0,0 +1,31 @@
...
+import sys
+
+import pytest
+from pytest import approx
+
+from brainscore_vision import score
+from brainscore_vision.utils import seed_everything
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+seed_everything(42)
```
Comment (mschrimpf) [inline brainscore_vision/models/vision_transformer/test.py:11]: move into function / delete

File: brainscore_vision/models/vision_transformer/test.py [hunk 34]
```diff
@@ -0,0 +1,31 @@
...
+
+
+@pytest.mark.private_access
```
Comment (mschrimpf) [inline brainscore_vision/models/vision_transformer/test.py:14]:
```suggestion
```

**Review by mschrimpf — CHANGES_REQUESTED** [review]
High-level points:
1. Just to make sure: all models have been vetted to reproduce existing scores on the website? Assuming yes, we should include this in the tests (can just add to the parametric tests with the private scores).
2. Why do we need to seed? Everything should be deterministic
3. move `torch.device` into the wrapper

**Comment by shehadak** [comment]
Great points, Martin.

> 1. Just to make sure: all models have been vetted to reproduce existing scores on the website? Assuming yes, we should include this in the tests (can just add to the parametric tests with the private scores).

You are correct. For models that do appear on the website, I made sure that the scores match. The most recent commits also include additional tests on private benchmarks.

> 2. Why do we need to seed? Everything should be deterministic

Seeding is necessary for models that generally use randomness during inference (e.g. untrained models with random initialization, VAEs, MC dropout), resulting in different scores even for the same benchmark run. I also think, generally, it's a good practice to seed test running for consistency. None of the models in this PR use it, so I removed seeding for this PR, but I think it still makes sense to include it in #498.

> 3. move `torch.device` into the wrapper

Great catch. Removed `torch.device` since it's handled by the wrapper anyway.

File: brainscore_vision/models/alexnet/model.py (hunk 1 above)
Comment (shehadak) [reply brainscore_vision/models/alexnet/model.py]: You're correct. It already exists in the Pytorch wrapper, so no need for it here.

File: brainscore_vision/models/alexnet/test.py (hunk 2 above)
Comment (shehadak) [reply brainscore_vision/models/alexnet/test.py]: Seeding is necessary for models that generally use randomness during inference (e.g. untrained models with random initialization, VAEs, MC dropout), resulting in different scores even for the same benchmark run. I also think, generally, it's a good practice to seed test running for consistency. None of the models in this PR use it, so I removed seeding for this PR.

File: brainscore_vision/models/crossvit_pretrained/model.py (hunk 5 above)
Comment (shehadak) [reply brainscore_vision/models/crossvit_pretrained/model.py:22]: The original model submission (`cv_18_dagger_408_pretrained`) from braintree.

File: brainscore_vision/models/crossvit_pretrained/model.py [hunk 35]
```diff
@@ -0,0 +1,59 @@
...
+def load_preprocess_custom_model(image_filepaths, image_size, **kwargs):
+    images = load_images(image_filepaths)
+    images = preprocess_images(images, image_size=image_size, **kwargs)
+    return images
+
+
+def get_model():
+    # Generate Model
+    model = create_model("crossvit_18_dagger_408", pretrained=True)
+    model.eval()
```
Comment (mschrimpf) [inline brainscore_vision/models/crossvit_pretrained/model.py:40]: I would also move this to the wrapper

File: brainscore_vision/models/crossvit_pretrained/model.py (hunk 7 above)
Comment (mschrimpf) [reply brainscore_vision/models/crossvit_pretrained/model.py:51]: bump

File: brainscore_vision/models/densenet/test.py [hunk 36]
```diff
@@ -0,0 +1,33 @@
...
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+
+
+@pytest.mark.travis_slow
+@pytest.mark.memory_intense
+@pytest.mark.parametrize("model_identifier, benchmark_identifier, expected_score", [
+    # Private
+    pytest.param("densenet121", "MajajHong2015.IT-pls", approx(0.5327, abs=0.0005), marks=[pytest.mark.private_access]),
+    pytest.param("densenet161", "MajajHong2015.IT-pls", approx(0.5445, abs=0.0005), marks=[pytest.mark.private_access]),
+    pytest.param("densenet169", "MajajHong2015.IT-pls", approx(0.5462, abs=0.0005), marks=[pytest.mark.private_access]),
```
Comment (mschrimpf) [inline brainscore_vision/models/densenet/test.py:18]: this is different from the website (`.548`)

File: brainscore_vision/models/efficientnet/test.py [hunk 37]
```diff
@@ -0,0 +1,41 @@
...
+@pytest.mark.memory_intense
+@pytest.mark.parametrize("model_identifier, benchmark_identifier, expected_score", [
+    # Private
+    pytest.param("efficientnet_b0", "MajajHong2015.IT-pls", approx(0.526, abs=0.0005), marks=[pytest.mark.private_access]),
+    pytest.param("efficientnet_b1", "MajajHong2015.IT-pls", approx(0.538, abs=0.0005), marks=[pytest.mark.private_access]),
+    pytest.param("efficientnet_b2", "MajajHong2015.IT-pls", approx(0.529, abs=0.0005), marks=[pytest.mark.private_access]),
+    pytest.param("efficientnet_b3", "MajajHong2015.IT-pls", approx(0.511, abs=0.0005), marks=[pytest.mark.private_access]),
+    pytest.param("efficientnet_b4", "MajajHong2015.IT-pls", approx(0.514, abs=0.0005), marks=[pytest.mark.private_access]),
+    pytest.param("efficientnet_b5", "MajajHong2015.IT-pls", approx(0.528, abs=0.0005), marks=[pytest.mark.private_access]),
+    pytest.param("efficientnet_b6", "MajajHong2015.IT-pls", approx(0.526, abs=0.0005), marks=[pytest.mark.private_access]),
```
Comment (mschrimpf) [inline brainscore_vision/models/efficientnet/test.py:22]: different from website (`.541`), is this the same model? https://www.brain-score.org/model/vision/665

File: brainscore_vision/models/efficientnet/test.py [hunk 38]
```diff
@@ -0,0 +1,41 @@
...
+    pytest.param("efficientnet_b7", "MajajHong2015.IT-pls", approx(0.535, abs=0.0005), marks=[pytest.mark.private_access]),
```
Comment (mschrimpf) [inline brainscore_vision/models/efficientnet/test.py:23]: different from website (`.543`) https://www.brain-score.org/model/vision/863

File: brainscore_vision/models/efficientnet/test.py [hunk 39]
```diff
@@ -0,0 +1,41 @@
...
+from brainscore_vision import score
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+
+
+@pytest.mark.travis_slow
+@pytest.mark.memory_intense
+@pytest.mark.parametrize("model_identifier, benchmark_identifier, expected_score", [
+    # Private
+    pytest.param("efficientnet_b0", "MajajHong2015.IT-pls", approx(0.526, abs=0.0005), marks=[pytest.mark.private_access]),
```
Comment (mschrimpf) [inline brainscore_vision/models/efficientnet/test.py:16]: different from website (`.547`) https://www.brain-score.org/model/vision/662

File: brainscore_vision/models/efficientnet/test.py [hunk 40]
```diff
@@ -0,0 +1,41 @@
...
+    pytest.param("efficientnet_b1", "MajajHong2015.IT-pls", approx(0.538, abs=0.0005), marks=[pytest.mark.private_access]),
+    pytest.param("efficientnet_b2", "MajajHong2015.IT-pls", approx(0.529, abs=0.0005), marks=[pytest.mark.private_access]),
```
Comment (mschrimpf) [inline brainscore_vision/models/efficientnet/test.py:18]: different from website (`.547`) https://www.brain-score.org/model/vision/663

File: brainscore_vision/models/efficientnet/test.py [hunk 41]
```diff
@@ -0,0 +1,41 @@
...
+    pytest.param("efficientnet_b3", "MajajHong2015.IT-pls", approx(0.511, abs=0.0005), marks=[pytest.mark.private_access]),
+    pytest.param("efficientnet_b4", "MajajHong2015.IT-pls", approx(0.514, abs=0.0005), marks=[pytest.mark.private_access]),
```
Comment (mschrimpf) [inline brainscore_vision/models/efficientnet/test.py:20]: different from website (`.543`) https://www.brain-score.org/model/vision/664

File: brainscore_vision/models/pixels/model.py (hunk 22 above)
Comment (mschrimpf) [reply brainscore_vision/models/pixels/model.py:24]: bump

File: brainscore_vision/models/pixels/requirements.txt (hunk 25 above)
Comment (mschrimpf) [reply brainscore_vision/models/pixels/requirements.txt:2]: bump

File: brainscore_vision/models/resnet/test.py [hunk 42]
```diff
@@ -0,0 +1,31 @@
...
+from brainscore_vision import score
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+
+
+@pytest.mark.travis_slow
+@pytest.mark.memory_intense
+@pytest.mark.parametrize("model_identifier, benchmark_identifier, expected_score", [
+    # Private
+    pytest.param("resnet-18", "MajajHong2015.IT-pls", approx(0.527, abs=0.0005), marks=[pytest.mark.private_access]),
```
Comment (mschrimpf) [inline brainscore_vision/models/resnet/test.py:16]: different from website (`.531`) https://www.brain-score.org/model/vision/620

File: brainscore_vision/models/resnet/test.py [hunk 43]
```diff
@@ -0,0 +1,31 @@
...
+    pytest.param("resnet-34", "MajajHong2015.IT-pls", approx(0.516, abs=0.0005), marks=[pytest.mark.private_access]),
```
Comment (mschrimpf) [inline brainscore_vision/models/resnet/test.py:17]: different from website (`.474`) https://www.brain-score.org/model/vision/621

File: brainscore_vision/models/resnet/test.py [hunk 44]
```diff
@@ -0,0 +1,31 @@
...
+    pytest.param("resnet-50", "MajajHong2015.IT-pls", approx(0.523, abs=0.0005), marks=[pytest.mark.private_access]),
```
Comment (mschrimpf) [inline brainscore_vision/models/resnet/test.py:18]: different from website (`.518`) https://www.brain-score.org/model/vision/622

File: brainscore_vision/models/squeezenet/test.py [hunk 45]
```diff
@@ -0,0 +1,29 @@
...
+from brainscore_vision import score
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+
+
+@pytest.mark.travis_slow
+@pytest.mark.memory_intense
+@pytest.mark.parametrize("model_identifier, benchmark_identifier, expected_score", [
+    # Private
+    pytest.param("squeezenet1_0", "MajajHong2015.IT-pls", approx(0.518, abs=0.001), marks=[pytest.mark.private_access]),
```
Comment (mschrimpf) [inline brainscore_vision/models/squeezenet/test.py:16]: different from website (`.459`) https://www.brain-score.org/model/vision/618

File: brainscore_vision/models/squeezenet/test.py [hunk 46]
```diff
@@ -0,0 +1,29 @@
...
+    pytest.param("squeezenet1_1", "MajajHong2015.IT-pls", approx(0.504, abs=0.001), marks=[pytest.mark.private_access]),
```
Comment (mschrimpf) [inline brainscore_vision/models/squeezenet/test.py:17]: different from website (`.457`) https://www.brain-score.org/model/vision/619

File: brainscore_vision/models/vision_transformer/test.py [hunk 47]
```diff
@@ -0,0 +1,33 @@
...
+from brainscore_vision import score
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+
+
+@pytest.mark.travis_slow
+@pytest.mark.memory_intense
+@pytest.mark.parametrize("model_identifier, benchmark_identifier, expected_score", [
+    # Private
+    pytest.param("vit_b_16", "MajajHong2015.IT-pls", approx(0.438, abs=0.001), marks=[pytest.mark.private_access]),
```
Comment (mschrimpf) [inline brainscore_vision/models/vision_transformer/test.py:16]: different from website (`.380`) https://www.brain-score.org/model/vision/735

File: brainscore_vision/models/vision_transformer/test.py [hunk 48]
```diff
@@ -0,0 +1,33 @@
...
+    pytest.param("vit_b_32", "MajajHong2015.IT-pls", approx(0.482, abs=0.001), marks=[pytest.mark.private_access]),
```
Comment (mschrimpf) [inline brainscore_vision/models/vision_transformer/test.py:17]: different from website (`.417`) https://www.brain-score.org/model/vision/736

File: brainscore_vision/models/vision_transformer/test.py [hunk 49]
```diff
@@ -0,0 +1,33 @@
...
+    pytest.param("vit_l_16", "MajajHong2015.IT-pls", approx(0.534, abs=0.001), marks=[pytest.mark.private_access]),
```
Comment (mschrimpf) [inline brainscore_vision/models/vision_transformer/test.py:18]: different from website (`.487`) https://www.brain-score.org/model/vision/733

File: brainscore_vision/models/vision_transformer/test.py [hunk 50]
```diff
@@ -0,0 +1,33 @@
...
+    pytest.param("vit_l_32", "MajajHong2015.IT-pls", approx(0.5, abs=0.001), marks=[pytest.mark.private_access]),
```
Comment (mschrimpf) [inline brainscore_vision/models/vision_transformer/test.py:19]: different from website (`.439`) https://www.brain-score.org/model/vision/737 (or `.454` https://www.brain-score.org/model/vision/734)

**Review by mschrimpf — CHANGES_REQUESTED** [review]
The code all looks good (aside from a few minor suggestions from the last review), but I am concerned about the apparent mismatch in scores for some of the models. In particular, `densenet169`, all `efficientnet`s, all `resnet`s, all `squeezenet`s and all `ViT`s have substantial mismatches. What is going on there?

**Pull request closed without merging**